artman/cli/main2.py

```diff
@@ -74,6 +76,36 @@ def main(*args):
         _run_artman_in_docker(flags)
 
 
+def _adjust_input_dir(input_dir):
+    # Adjust input directory to use versioned shared config under
+    # /googleapis/gapic/[core,lang,packaging]. Currently, the codegen has
+    # coupling with some shared configuration yaml under those directories,
+    # causing library generation to fail when a breaking change is made to such
+    # shared configuration file. This delivers a poor user experience to artman
+    # users, as their library generation could fail without any change at API
+    # proto side.
+    # TODO(ethanbao): Remove this logic once
+    # https://github.com/googleapis/toolkit/issues/1450 is fixed.
+    if os.getenv(RUNNING_IN_ARTMAN_DOCKER_TOKEN):
+        # Only doing this when running inside Docker container
+        DIRS = [
```

Comment:
This is not a module constant, so rename to snake_case (`common_config_dirs`?)

Reply:
Done.